charts/argo/templates/server-cluster-roles.yaml

```diff
@@ -111,11 +111,7 @@ rules:
   - delete
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-{{- if .Values.singleNamespace }}
-kind: Role
-{{- else }}
 kind: ClusterRole
```

Comment:
Will also need to fix the bindings in https://github.com/argoproj/argo-helm/blob/556011dd4d764eb8c11a8d41eb9526e67e6c186e/charts/argo/templates/server-crb.yaml#L23

Reply:
Done.